jujugui/static/gui/src/app/components/env-switcher/env-switcher.js

```diff
@@ -65,9 +65,10 @@ YUI.add('env-switcher', function() {
         updated.
       @param {Object} data The data from the listEnvs call.
     */
-    updateEnvListCallback: function(callback, data) {
-      if (data.err) {
-        console.log(data.err);
+    updateEnvListCallback: function(callback, error, data) {
+      var err = data.err || error;
```

Comment:
This is because of the different response signatures of jem vs jes right? Could you add a comment to that effect.